### PR TITLE
Bobcat upgrade

### PR DESCRIPTION
**Pull request by wiiitek**
## Description
Upgrade to newest released version of Bobcat and dependencies clean-up.

## Motivation and Context
We should review our functional tests and make them run faster. They might be more compact.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: integration-tests/sanity-functional/src/main/resources/config/common/webdriver.properties [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+webdriver.defaultTimeout=10
+page.title.timeout=30
+thread.count=3
+
+
+# +++ Chrome +++
+#webdriver.chrome.driver property should point to Chrome webdriver executable.
+webdriver.type=chrome
+webdriver.chrome.driver=D:/Selenium/chromedriver.exe
```
Comment (Skejven) [inline integration-tests/sanity-functional/src/main/resources/config/common/webdriver.properties]: When this path is required. Will this work on all instances?

Comment (wiiitek) [reply integration-tests/sanity-functional/src/main/resources/config/common/webdriver.properties]: You are right: this is an arbitrary value. If chromedriver is not found then there is a stacktrace with following error:

```
2) Error in custom provider, java.lang.IllegalStateException: The driver executable does not exist: D:\Selenium\chromedriver.exe
```

If this config value is commented then we have following error:

```
2) Error in custom provider, java.lang.IllegalStateException: The path to the driver executable must be set by the webdriver.chrome.driver system property;
```

What should we do here? Currently on master we have: 

* [integration-tests/sanity-functional/src/main/config/webdriver.properties#L23](https://github.com/Cognifide/aet/blob/master/integration-tests/sanity-functional/src/main/config/webdriver.properties#L23)

Comment (Skejven) [reply integration-tests/sanity-functional/src/main/resources/config/common/webdriver.properties]: Maybe we should at least document the necessary change on the path to the chromedriver in the comment on in the [it-tests README](https://github.com/Cognifide/aet/tree/master/integration-tests#sanity-functional) ?

Comment (wiiitek) [reply integration-tests/sanity-functional/src/main/resources/config/common/webdriver.properties]: I have update the README

**Pull request merged**